backend/tests/unit/dal/Services/ProjectServiceTest.cs

```diff
@@ -389,7 +390,31 @@ public void Get_Sensitive_AsAdmin()
         }
         #endregion
 
-        #region Add
+        /// <summary>
+        /// User with appropriate permission successfully adds new project. Project Number is auto-generated.
+        /// </summary>
+        #region Add 
+        [Fact]
+        public void Add_Project()
+        {
+            // Arrange
+            var helper = new TestHelper();
+            var user = PrincipalHelper.CreateForPermission(Permissions.PropertyAdd).AddAgency(1);
+            var project = EntityHelper.CreateProject(1);
```

Comment:
Now you may want to prove that the number is generated by starting with a different number here.

Reply:
Will add that in before I merge!